librz/bin/bin.c

```diff
@@ -104,11 +104,14 @@ RZ_API RzList *rz_bin_dump_strings(RzBinFile *bf, int min, int raw) {
 	return rz_bin_file_get_strings(bf, min, 1, raw);
 }
 
-RZ_API void rz_bin_options_init(RzBinOptions *opt, int fd, ut64 baseaddr, ut64 loadaddr, bool patch_relocs, int rawstr) {
+RZ_API void rz_bin_options_init(RzBinOptions *opt, int fd, ut64 baseaddr, ut64 loadaddr, bool patch_relocs, bool elf_load_sections, bool elf_checks_sections, bool elf_checks_segments, int rawstr) {
```

Comment:
As I wrote in the comment I would not put these as parameters of this function, but just set them outside of it when necessary.